.github/workflows/test.yml

```
@@ -15,11 +15,83 @@ jobs:
+        with:
+          image: ${{ env.SUCCESS_IMAGE }}
+          config-file: /non/existent/path.yaml
+      # Ensures proper error reporting when an unsupported container
+      # runtime (not docker/podman) is specified
+      - name: 'Negative test: Invalid image source'
+        if: always() && steps.pre-setup.outcome == 'success'
+        continue-on-error: true
+        uses: ./
+        with:
+          image: ${{ env.SUCCESS_IMAGE }}
+          image-source: invalid-source
```

actionlint (1.7.4)

input "image-source" is not defined in action "Dive Action v2" defined at "./". available inputs are "config-file", "dive-image-registry", "dive-image-version", "github-token", "image"

(action)